examples/asset/asset_pack.rs

```diff
@@ -0,0 +1,34 @@
+//! Example of a custom asset bundle
+
+use bevy::asset::io::pack::{AssetPack, AssetPackPlugin, GetPack};
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins((DefaultPlugins, AssetPackExamplePlugin))
+        .add_systems(Startup, setup)
+        .run();
+}
+
+struct AssetPackExamplePlugin;
+
+#[derive(AssetPack)]
+struct ExampleAssetPack {
+    #[embedded("files/bevy_pixel_dark.png")]
+    sprite: Handle<Image>,
+}
+
+impl Plugin for AssetPackExamplePlugin {
+    fn build(&self, app: &mut App) {
+        app.add_plugins(AssetPackPlugin::<ExampleAssetPack>::default());
+    }
+}
```

Comment:
```suggestion
fn main() {
    App::new()
        .add_plugins(DefaultPlugins)
        .add_plugins(AssetPackPlugin::<ExampleAssetPack>::default())
        .add_systems(Startup, setup)
        .run();
}

#[derive(AssetPack)]
struct ExampleAssetPack {
    #[embedded("files/bevy_pixel_dark.png")]
    sprite: Handle<Image>,
}
```

No need for a plugin, let's keep this example short :)